### ISSUE TITLE
Timelines not loading (proxy problem?)

### ISSUE THREAD

**Issue by jimmykl** [NUKnightLab/TimelineJS3#657]
I'm trying to debug some `Timeline configuration has no events.` errors, you can see one here https://www.audioculture.co.nz/scenes/aotearoa-punk-timeline This was working until quite recently and nothing was changed on the site.

The spreadsheet link is: https://docs.google.com/spreadsheets/d/1NjZWR5xS7g6faFeXsipowPmG3JaP-r1_tFrpGNo0I7Y/edit#gid=0

From the console it looks like this isn't loading https://sheets-proxy.knightlab.com/proxy/https://docs.google.com/spreadsheets/d/1NjZWR5xS7g6faFeXsipowPmG3JaP-r1_tFrpGNo0I7Y/pub?output=csv with an error of `not valid for proxy`

**Comment by jimmykl** [comment]
I found some errors with line breaks in the CSV (https://docs.google.com/spreadsheets/d/1NjZWR5xS7g6faFeXsipowPmG3JaP-r1_tFrpGNo0I7Y/pub?output=csv) which I resolved in the Google Sheet but the proxy is still returning not valid…

**Comment by JoeGermuska** [comment]
I just loaded https://www.audioculture.co.nz/scenes/aotearoa-punk-timeline and the timeline is working!

Are you still getting the error?

We did some things to try to lock the proxy down out of concern that other people faced with this change from Google would take advantage of it. It's already under heavy load trying to serve existing timeline traffic, and we can't afford to run a general purpose proxy.

It would have been easy to just use the referrer for the Timeline host page (knightlab == ok), we want to support those who, like you, have instantiated timelines by using the javascript code directly, instead of embedding the iframe. 

It should be OK as long as the columns in the proxied spreadsheet are the same as or a subset of the official column headers -- so whitespace and the like could have been a problem not just with #656 but also with the proxy.  I'll consider stripping whitespace before evaluating the headers, but as noted in #656, it will fail later if not sooner.  Still working on getting better messaging when the proxy says ⛔
